docs/get_started/tour.mdx

```diff
@@ -45,1015 +51,828 @@ This guide is written for:
-The concurrency guarantees of keyed services simplify reasoning about interaction with external systems, because you are sure that there is only a single ongoing invocation for a single key.
-For example, imagine you have a keyed service which is the single writer to a database and every key only interacts with an isolated set of database rows.
-Then you can scale your application and never have concurrent invocations writing to the same database row.
-This resolves common data consistency issues such as lost updates or non-repeatable reads.
-
-This also means that time-consuming operations in a keyed service, for example sleeps, lock that key/service for the entire operation.
-Make sure you don't introduce cyclical request-response calls in your application.
-For example, if A calls B, and B calls C, and C calls A again.
-Each service waits on the response of the other and none of them can progress.
-The keys remain locked and the system can't process any more requests.
-</Admonition>
-
-## Service calls
-
-<Admonition type="note" title="Implement it yourself or follow along by looking at the code under `part1`"/>
-
-One of the key parts of distributed applications is service-to-service communication.
-Calls are implemented as RPC messages.
-Restate makes service-to-service communication reliable by ensuring that messages do not get lost.
 
-### Reliable message sending without queues
+## Invoking Handlers
+<Admonition type="note" title="Implement it yourself or follow along by looking at the code under part1"/>
 
-The tour begins with sending messages between services.
+Handlers can be invoked in several ways: via HTTP requests, programmatically with the SDK, or via Kafka events.
 
-In this example, when you add a seat to your shopping cart, it gets reserved for 15 minutes.
-When a user didn't proceed with the payment before the timeout, you need to call the `UserSession/expireTicket` function.
-Let the `expireTicket` function call the `TicketService/unreserve` function.
-
-<Tabs groupId="sdk" queryString>
-<TabItem value="ts" label="TypeScript">
-
-```typescript ticket_service.ts
-CODE_LOAD::https://raw.githubusercontent.com/restatedev/examples/main/tutorials/tour-of-restate-typescript/src/part1/user_session.ts#<start_expire_ticket>-<end_expire_ticket>
-```
+### Request-response calls over HTTP
+Let's start with invoking our handler over HTTP using `curl`.
 
-Specify that you want to call the `TicketService` by supplying `ticketServiceApi` to the `send` function.
-Then call the `unreserve` function on the `TicketService`.
-
-Once you have added this to the code, try it out by calling the `UserSession/expireTicket` function:
+For example, add a ticket `seat2B` to the cart of Mary by calling the `addTicket` handler of the `CartObject`:
 
 ```shell
-curl localhost:8080/UserSession/expireTicket \
-    -H 'content-type: application/json' \
-    -d '{"key": "123", "request": "456"}'
+curl localhost:8080/CartObject/Mary/addTicket -d '"seat2B"'
```

Comment:
To me this command is less confusing if you add the content type header. Also it shouldn't work like it is written here...